pkg/products/grafana/reconciler.go

```diff
@@ -637,6 +244,16 @@ func (r *Reconciler) deleteConsoleLink(ctx context.Context, serverClient k8sclie
 	return nil
 }
 
-func boolPtr(value bool) *bool {
-	return &value
+func (r *Reconciler) deleteMonitoringOperatorNamespace(ctx context.Context, serverClient k8sclient.Client, installation *integreatlyv1alpha1.RHMI) error {
+	nsPrefix := installation.Spec.NamespacePrefix
+	ns := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: nsPrefix + "customer-monitoring-operator",
+		},
+	}
+	err := serverClient.Delete(ctx, ns)
```

Comment:
wdyt if we would first try find the ns and not attempt to delete it it doesn't exists - or, we could handle the deletion error.

Reply:
Seems to me no need change, as error is checked, and missing ns error is checked, and it's shorter, than check if ns exists (if yes - it's 2 steps, and in our case - just 1). I actually thought about check ns before implement this.
Thanks you for comment